oatomobile/baselines/torch/dim/train.py

```diff
@@ -195,10 +204,15 @@ def train_step(
         is_at_traffic_light=batch["is_at_traffic_light"],
         traffic_light_state=batch["traffic_light_state"],
     )
-    _, log_prob, logabsdet = model._decoder._inverse(y=y, z=z)
 
-    # Calculates loss (NLL).
-    loss = -torch.mean(log_prob - logabsdet, dim=0)  # pylint: disable=no-member
+    if use_tcn:
```

Comment:
the API for the `decoder` should be the same, regardless if it's an RNN or a TCN

Reply:
Can you clarify what you mean by this? By API, do you want the TCN decoder to similarly have a `.inverse` method, as the `Autoregressive` decoder does?